File: python-vs-javascript/1-basics/7-dictionaries.py
#Dictionaries are a data structure that uses a key: value structure.
dict_obj = {"fname":"Richard","lname":"Hunt", "age":40, "gender":"M"}

#You access an element by calling the key.
print(dict_obj["lname"])

#To print to the console the keys.
for key in dict_obj:
    print(key)


#To get the values we need to used a slightly different approach.
for key, val in dict_obj.items():
    print(val)

File: python-vs-javascript/1-basics/7-dictionaries.js
//Dictionaries are a data structure that uses a key: value structure.
let dictObj = {fname:"Richard",lname:"Hunt", age:40, gender:"M"};

//You access an element by calling the key.
console.log(dictObj.lname);

//To print to the console the keys.
for (var key in dictObj){
    console.log(key);
}

//To get the values we need to used a slightly different approach.
for (var key in dictObj){
    console.log(dictObj[key]);
}